optimum/intel/neural_compressor/quantization.py

```diff
@@ -320,7 +324,23 @@ def from_pretrained(
 
                     raise EnvironmentError(msg)
 
-            state_dict = torch.load(state_dict_path)
+            try:
+                state_dict = torch.jit.load(state_dict_path)
+            except:
+                state_dict = torch.load(state_dict_path)
+        if isinstance(state_dict, torch.jit._script.RecursiveScriptModule):
+            assert example_inputs, "Please provide the example_inputs for IncQuantizedModel loading."
```

Comment:
Can you raise an exception instead ? (to keep consistency)

Reply:
Done